apis/nucleus/src/components/listbox/components/ListBoxRowColumn/ListBoxRowColumn.jsx

```diff
@@ -91,32 +92,25 @@ function RowColumn({ index, rowIndex, columnIndex, style, data }) {
 
   const handleKeyDownCallback = useCallback(getFieldKeyboardNavigation({ ...actions, focusListItems }), [actions]);
 
-  const [isSelected, setSelected] = useState(false);
-  const [cell, setCell] = useState();
+  const getCellFromPagesCallback = useCallback(() => getCellFromPages({ pages, cellIndex }), [pages, cellIndex]);
 
+  const [cell, setCell] = useState();
```

Comment:
would
```suggestion
  const cell = useMemo(() => getCellFromPages({ pages, cellIndex }), [pages, cellIndex]);
```
work?

Reply:
I will try

Reply:
Yes, it seems to work 🎉  I removed the effect as well.